tests/test_toggle.py

```diff
@@ -38,18 +38,28 @@ def test_report(self):
         self.assertTrue(
             "Toggle: USE_PYSIDE=" in modmesh.Toggle.instance.report())
 
-    def test_static_constexpr(self):
-        self.assertTrue(hasattr(modmesh.Toggle, "USE_PYSIDE"))
-        self.assertTrue(hasattr(modmesh.Toggle.instance, "USE_PYSIDE"))
-
     def test_instance(self):
-        self.assertTrue(hasattr(modmesh.Toggle.instance, "show_axis"))
+        self.assertTrue(hasattr(modmesh.Toggle.fixed, "use_pyside"))
+        self.assertTrue(hasattr(modmesh.Toggle.fixed, "show_axis"))
+
+    def test_clone(self):
+        tg = modmesh.Toggle.instance.clone()
+        tg.dynamic_clear()
+        self.assertEqual(tg.dynamic_keys(), [])
+
+        tg.set_bool("test_bool", True)
+        self.assertTrue(tg.get_bool("test_bool"))
+        self.assertEqual(tg.dynamic_keys(), ["test_bool"])
+
+        tg1 = tg.clone()
+        self.assertEqual(tg.dynamic_keys(), tg1.dynamic_keys())
+        self.assertEqual(tg.get_bool("test_bool"), tg1.get_bool("test_bool"))
 
 
 class ToggleDynamicTC(unittest.TestCase):
 
     def test_all_types(self):
-        tg = modmesh.Toggle.instance
+        tg = modmesh.Toggle.instance.clone()
```

Comment:
Clone the `Toggle` singleton so that the test code does not accidentally change default settings (if any, although there are few at the moment).